MediaBrowser.Providers/Plugins/TheTvdb/TvdbSeriesProvider.cs

```diff
@@ -251,10 +244,10 @@ private async Task<List<RemoteSearchResult>> FindSeriesInternal(string name, str
                     SearchProviderName = Name
                 };
 
-                if (!string.IsNullOrEmpty(seriesSearchResult.Banner))
+                if (!string.IsNullOrEmpty(seriesSearchResult.Poster))
```

Comment:
Why change from Banner to Poster?

Reply:
Banners are wide and horizontal while search result images should be portrait (posters are). I don't know why it was ever set to `Banner` in the first place. Probably nobody noticed it because search was never working properly.